lib/twilio.js

```diff
@@ -0,0 +1,39 @@
+const twilio = require('twilio')
+
+const helpers = require('./helpers.js')
+
+// Set up Twilio
+const TWILIO_SID = helpers.getEnvVar('TWILIO_SID')
+const TWILIO_TOKEN = helpers.getEnvVar('TWILIO_TOKEN')
+const twilioClient = twilio(TWILIO_SID, TWILIO_TOKEN)
+const MessagingResponse = twilio.twiml.MessagingResponse
+
+async function sendTwilioMessage(toPhoneNumber, fromPhoneNumber, message) {
+    try {
+        const response = await twilioClient.messages.create({
+            to: toPhoneNumber,
+            from: fromPhoneNumber,
+            body: message,
+        })
+
+        helpers.log(response.sid)
+
+        return response
+    } catch(err) {
+        helpers.log(err.toString())
+
+        return
+    }
+}
+
+async function sendTwilioResponse(response, message) {
+    let twiml = new MessagingResponse()
```

Comment:
I'm not convinced that we need to use `twiml` for responding to messages. It might be easier just to use the one `sendTwilioMessage` function for both new messages and for replies. What do you think?

Reply:
No preference / opinion on my end.

Reply:
I will leave it as is for now. Thanks!